blogs/whats_new_1.7/index.md

```diff
@@ -0,0 +1,153 @@
+
+- The eBPF flow filter allows simple filtering at the core level where it decides what flows to collect.  Here are the changes.
+    - `ports`, `sourcePorts` or `destPorts`: Supports up to two TCP, UDP or SCTP ports.  Previously, it allowed only one port.
+    - If `ports` is specified but not `protocol`, it filters on TCP and UDP flows.
+    - `tcpFlags`: Filter the flows with this TCP flag.  It must be one of "SYN", "SYN-ACK", "ACK", "FIN", "RST", "URG", "ECE", "CWR", "FIN-ACK", or "RST-ACK".
+
+- The Kubernetes service for DNS does a reverse proxy and maps to an internal port at the pod running DNS.  For the DNS Tracking feature to work end-to-end, specify the internal port value using the environment variable `DNS_TRACKING_PORT`.
+
+Figure 4 shows an example use-case scenario that only collects DNS traffic on ports 53 and 5353.  OVN, which is the default CNI for OpenShift, maps the internal `DNS_TRACKING_PORT` to 5353.  If the DNS Tracking feature is enabled, it displays data for DNS latency, response code, and error on a per-flow basis in the Traffic flows table.
+```
+  agent:
+    ebpf:
+      flowFilter:
+        enable: true
+        ports: 53,5353  # only have flows for these two ports
+      advanced:
+        env:
+          DNS_TRACKING_PORT: "5353"  # must be a string
+      sampling: 1
+      features:
+        - DNSTracking
+```
+<p style="text-align: center">Figure 4: eBPF flow filter for DNS traffic</p>
+
+Assuming you have **oc** installed on your computer, enter `oc edit flowcollector`.  Look for the `ebpf` section and add the lines in Figure 4.
+
+Figure 5 shows an example where the eBPF Agent collects only TCP flows that have the SYN flag, which are flows that initiate a TCP connection.  By not specifying `protocol`, it will also collect UDP.
+
+```
+  agent:
+    ebpf:
+      flowFilter:
+        enable: true
+        tcpFlags: SYN
+      sampling: 1
+```
+<p style="text-align: center">Figure 5: eBPF flow filter for TCP SYN and UDP traffic</p>
+
+In the Traffic flows table, you can add the **TCP flags** column.  You can also filter on this column (Figure 6).
+
+![Traffic flows - TCP flags](images/traffic_flows-tcp_flags.png)
+<div style="text-align: center">Figure 6: Traffic flows - TCP flags</div>
+
+
+## Network Observability in Developer perspective
+
+For the first time, NetObserv is providing observability in the Developer perspective!  If you are in Administrator perspective, click **Administrator** in the upper left corner and select **Developer**.  Click **Project** and select a project in the drop down near the top of the page.  In the **Inventory** section, click **Pods** or **Service**.   Click a particular item on a row.  There will be a **Network Traffic** tab.  Click this link to bring up the familiar page with *Overview*, *Traffic flows*, and *Topology* tabs filtered to this pod or service (Figure 7).
```

Comment:
Should we mention the netobserv-reader & netobserv-metrics-reader roles here ?

https://docs.openshift.com/container-platform/4.17/observability/network_observability/installing-operators.html#network-observability-multi-tenancy_network_observability

Reply:
I added a sentence to refer to the documentation.